sensing/autoware_pointcloud_preprocessor/src/filter.cpp

```diff
@@ -70,14 +70,14 @@ autoware::pointcloud_preprocessor::Filter::Filter(
 {
   // Set parameters (moved from NodeletLazy onInit)
   {
-    tf_input_frame_ = static_cast<std::string>(declare_parameter("input_frame", ""));
-    tf_output_frame_ = static_cast<std::string>(declare_parameter("output_frame", ""));
-    max_queue_size_ = static_cast<std::size_t>(declare_parameter("max_queue_size", 5));
+    tf_input_frame_ = declare_parameter<std::string>("input_frame");
+    tf_output_frame_ = declare_parameter<std::string>("output_frame");
+    max_queue_size_ = declare_parameter<int64_t>("max_queue_size");
```

Comment:
Same as other PRs. Implicit cast -> static cast
There are initializations in the header that are only noise given how parameters are loaded. Since we are workin on this now, could you delete those?

Reply:
Thanks!
static cast: https://github.com/autowarefoundation/autoware.universe/pull/8469/commits/c338a59d9424b6c8413a3383204ba73d3826fe83

remove init value:  https://github.com/autowarefoundation/autoware.universe/pull/8469/commits/9819fb6273683f2963e1b0cfd0070e851dfa22ac